docs/Nvidia.md

```diff
@@ -0,0 +1,18 @@
+# NVIDIA drivers
```

Comment:
Maybe I'm missing it, but it doesn't look like this page is actually used anywhere.